test/images/pets/zookeeper-installer/install.sh

```diff
@@ -55,10 +55,10 @@ cp "${INSTALL_VOLUME}"/zookeeper/conf/zoo_sample.cfg "${INSTALL_VOLUME}"/zookeep
 
 # TODO: Should dynamic config be tied to the version?
 IFS="." read -ra RELEASE <<< "${VERSION}"
-if [ $(expr "${RELEASE[1]}") -gt 4 ]; then
+if [ "$(("${RELEASE[1]}"))" -gt 4 ]; then
```

Comment:
Why I do this change:

```
In test/images/pets/zookeeper-installer/install.sh line 58:
if [ $(expr "${RELEASE[1]}") -gt 4 ]; then
     ^-- SC2046: Quote this to prevent word splitting.
       ^-- SC2003: expr is antiquated. Consider rewriting this using $((..)), ${} or [[ ]].
```